x-pack/platform/plugins/shared/fleet/server/services/epm/packages/remove.ts

```diff
@@ -427,15 +447,18 @@ export async function deleteKibanaSavedObjectsAssets({
     .map(({ id, type }) => ({ id, type } as KibanaAssetReference));
 
   try {
-    const packageInfo = await Registry.fetchInfo(
-      installedPkg.attributes.name,
-      installedPkg.attributes.version
-    );
+    const packageInfo = await getPackageInfo({
+      savedObjectsClient,
+      pkgName: installedPkg.attributes.name,
+      pkgVersion: installedPkg.attributes.version,
+      skipArchive: installedPkg.attributes.install_source !== 'registry',
+    });
```

Comment:
Replacing it also here, it is a different code path but this way we ensure that packages that it doesn't error for packages not installed from registry.